tests/Agent/IntegrationTests/UnboundedServices/docker-compose.yml

```diff
@@ -133,24 +133,43 @@ services:
         container_name: MySqlServer
 
     elastic:
-        image: elastic/elasticsearch:8.6.2
+        image: elasticsearch:8.6.2
         ports:
-            - "9200:9200"
-            - "9300:9300"
+            - 9200:9200
         environment:
             - discovery.type=single-node
-            - xpack.security.enabled=false
-            - xpack.security.enrollment.enabled=false
+            - ELASTIC_PASSWORD=${ELASTIC_PASSWORD:-ElasticPassword}
+            - xpack.security.enabled=true
             - xpack.security.http.ssl.enabled=false
-            - xpack.security.transport.ssl.enabled=false
+        mem_limit: 2g
```

Comment:
I don't think so; that config was intended to work around a warning from Elastic at startup, complaining about not having enough virtual memory areas. But I just checked again and still get the warning with or without the memory limit.  I'll remove it since it doesn't really matter. 

If you're interested in the virtual memory area warning, [this is the only supported way to completely fix it on Docker Desktop](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_windows_with_docker_desktop_wsl_2_backend) - and I *really* don't want to make everyone set a magic configuration for their local environment if we don't have to...